example/email_action_delivered.json

```diff
@@ -0,0 +1,31 @@
+{
+  	"meta": {
+		"topic": [
+			"marketing"
+		]
+	},
+	"type": "DeliveryEvent",
+	"agent": {
+	  "type": "Person",
```

Comment:
is Brian the agent owner of the marketing program?  In reflex, the URI for the agent profile `https://{id}.bhhs.hsfaffiliates.com/profile/card#me` is all you need to include.   If that's not available let's discuss.

Reply:
@dduran1967 yes he'd be the owner.  I would need to confirm which id to use, do you have an example of a working url?